db/tikv/coprocessor.go

```diff
@@ -74,17 +73,76 @@ func (db *coprocessor) Read(ctx context.Context, table string, key string, field
 	if err != nil {
 		return nil, err
 	}
+
 	//return db.table.DecodeValue(result.GetData(), fields)
 	return make(map[string][]byte, len(fields)), nil
 }
 
 func (db *coprocessor) Scan(ctx context.Context, table string, startKey string, count int, fields []string) ([]map[string][]byte, error) {
+	// construct the req
+	req := kv.Request{}
+	req.Concurrency = 1
+	req.Tp = kv.ReqTypeDAG
+	req.KeyRanges = []kv.KeyRange{db.table.GetScanRange(startKey)}
+	dag := db.table.BuildDAGTableScanWithLimitReq(fields, count)
+	req.StartTs = dag.StartTs
+	data, err := dag.Marshal()
+	if err != nil {
+		return nil, err
```

Comment:
maybe we can `import github.com/juju/errors` and:
```go
return nil, errors.Trace(err)
```

Reply:
no need error, we don't care the root cause in this repo.